### PR TITLE
[13.0] [MIG] website_sale_product_reference_displayed: Migration to v13

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT26955

please @chienandalu @joao-p-marques, review this

### CODE REVIEW

File: website_sale_product_reference_displayed/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Display product reference in e-commerce",
+    "version": "13.0.1.0.0",
+    "category": "Website",
+    "website": "https://github.com/OCA/e-commerce",
+    "author": "Tecnativa, " "Odoo Community Association (OCA)",
```
Comment (victoralmau) [inline website_sale_product_reference_displayed/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

**Review by victoralmau — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
:+1:

**Comment by CarlosRoca13** [comment]
@victoralmau change done

**Review by HaraldPanten — APPROVED** [review]
Seems to be OK

**Pull request merged**